dev/common/container_auth_private_key.pem

```diff
@@ -0,0 +1,5 @@
+-----BEGIN EC PRIVATE KEY-----
```

Comment:
These keys are just for the developer environment, so it's okay for them to get committed.